### ISSUE TITLE
CI test failure in Julia nightly due to Unicode changes

### ISSUE THREAD

**Issue by fingolfin** [oscar-system/Oscar.jl#1140]
CI logs say:
```
Printing: Test Failed at /home/runner/work/Oscar.jl/Oscar.jl/test/printing.jl:6
  Expression: Oscar.is_unicode_allowed() == true
   Evaluated: false == true
```
The subsequent errors are probably just a consequence of this.

Same test passes in Julia 1.6 and 1.7

**Comment by thofma** [comment]
I think it is a bug in Preferences.jl

**Comment by thofma** [comment]
https://github.com/JuliaPackaging/Preferences.jl/issues/31

**Comment by thofma** [comment]
This was fixed and will be available once a new release is made for Preferences.jl.

**Comment by fingolfin** [comment]
Once this is resolved we should revert PR #1156
